levels/LevelSelector.tscn

```diff
@@ -118,5 +119,4 @@ __meta__ = {
 
 [node name="Node2D" type="Node2D" parent="."]
 [connection signal="pressed" from="Obstacle/Button" to="Obstacle/Button" method="_on_Button_pressed"]
-[connection signal="pressed" from="Obstacle3/Button" to="Obstacle3/Button" method="_on_Button_pressed"]
```

Comment:
Removed the "Get more levels" button since the PAD UI is placed in the landing page.